.github/FUNDING.yml

```diff
@@ -0,0 +1,8 @@
+# These are supported funding model platforms
```

Comment:
not sure what's going on here, why is this being re-added in this pr?

Reply:
Probably automatically added from the github funding beta

Reply:
@Skillz4Killz

Reply:
@Skillz4Killz

Reply:
How do i get rid of this or should i remake pr or just leave it?